### PR TITLE
📊 Cholera - combining fasttrack and backport of cholera data

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

**Comment by Marigold** [comment]
@spoonerf do you have a sense what would it take to move GHO to ETL? We do have some [legacy WHO](https://github.com/owid/etl/blob/master/etl/steps/data/meadow/who/2021-07-01/gho.ipynb) in there, but I guess that's not very useful. Would it be enough to migrate it from [importers](https://github.com/owid/importers/blob/68e8a9b27b8c1ff7b008595d89fcef816a18f942/who_gho/main.py)? I'm working on WDI indicators right now and I assume this could be pretty similar, so perhaps I could give it a shot.

(I ask because my plan is to get rid of backported datasets)

**Comment by Marigold** [comment]
Makes a ton of sense @spoonerf. Getting rid of backports is not urgent (yet), it'd be just one less system to manage (so nice to have only) 🤞 they can release it soon.

**Comment by spoonerf** [comment]
Thanks for understanding! Let me know if it becomes more urgent and I will try and get a better sense of the WHO timeline.

**Review by pabloarosado — APPROVED** [review]
Looks good, except for a missing dependency. I made other suggestions, but feel free to ignore them.

**Comment by Marigold** [comment]
@pabloarosado I actually have a better way than fast-track which is `backport-migrate` CLI. You give it dataset ID and it generates code in ETL including metadata YAMLs, then you run chart revision tool and you're done (you don't even have to do that if you save it into the same namespace). That was my plan how to avoid backporting.

The problem with GHO is that it has several thousand variables and YAML file for that would be massive...

File: etl/steps/data/garden/who/2023-06-01/cholera.excluded_countries.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[
+    "Africa",
+    "Americas",
+    "Eastern Mediterranean",
+    "High-income",
+    "Lower-middle-income",
+    "South-east Asia",
+    "Upper-middle-income",
+    "Western Pacific",
+    "World"
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/who/2023-06-01/cholera.excluded_countries.json:10]: Strangely the regional and global data was only given for 2013, so I decided to remove it and recalculate the global total for all years. I am slightly unsure about adding in the regional data as the data is so patchy, but I suppose that is okay if we highlight that it is reported cases only.

**Pull request merged**